.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -19,12 +19,51 @@ body:
       description: If on self-hosted, what version are you running?
     validations:
       required: false
+  - type: dropdown
+    id: product-area
+    attributes:
+      label: Product Area
+      description: Where in the product did you find the bug?
+      options:
+        # Keep in sync with feature.yml and labels.yml
+        - 'Sign in'
+        - 'Issues'
+        - 'Projects'
+        - 'Performance'
+        - 'Profiling'
+        - 'Replays'
+        - 'Crons'
+        - 'Alerts'
+        - 'Discover'
+        - 'Dashboards'
+        - 'Releases'
+        - 'User Feedback'
+        - 'Stats'
+        - 'Settings'
+        - 'Settings - General'
+        - 'Settings - Projects'
+        - 'Settings - Teams'
+        - 'Settings - Members'
+        - 'Settings - Security & Privacy'
+        - 'Settings - Auth'
+        - 'Settings - Relay'
+        - 'Settings - Repositories'
+        - 'Settings - Integrations'
+        - 'Settings - Developer Settings'
+        - 'Settings - Spike Protection'
+        - 'Help'
+        - 'What’s new'
+        - 'Service status'
+        - 'Footer'
+        - 'Other'
+    validations:
+      required: true
```

Comment:
Do we need this to be required in the templates?

Reply:
It's intended to replace manual routing, so I'd like to require it, yes. We have an "Other" option if people want to be lazy or genuinely don't know.

Reply:
After talking to Support it sounds like they're still going to have the first touch, so dropping this back to optional since Support can fill it in if missing.